xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-test/xwiki-platform-menu-test-tests/src/test/it/org/xwiki/menu/test/ui/MenuInAdministrationTestIT.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.menu.test.ui;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+import org.xwiki.administration.test.po.AdministrationPage;
+import org.xwiki.appwithinminutes.test.po.EntryNamePane;
+import org.xwiki.menu.test.po.MenuEntryEditPage;
+import org.xwiki.menu.test.po.MenuHomePage;
+import org.xwiki.model.reference.DocumentReference;
+import org.xwiki.test.ui.AbstractTest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test to check that the Menu app is available through the administration and can be used to add new entries
+ *
+ */
+public class MenuInAdministrationTestIT extends AbstractTest
+{
+    @Test
+    public void verifyMenuIsAvailableInAdministration()
+    {
+        // Log in as superadmin
+        getUtil().login("superadmin", "pass");
```

Comment:
Does it work if you use the Admin user?

Reply:
There's no admin user :)

Reply:
(the admin user is in the flavor and creating a new user in each test is possible but a bit overkill - this is the current practice we're following)

Reply:
Ok that explain why it was used :)

Reply:
Would be better to use loginAsSuperAdmin(), the test should not have to care about the configured password.

Reply:
+1